Makefile

```diff
@@ -25,3 +25,6 @@ stan-baseline:
 
 lint-docker:
 	docker run -it --rm -v "$(CURDIR)":/home/node/app -w /home/node/app -u node node:20 npm install && npm run lint
+
+jest:
+	npm install && npm run test
```

Comment:
Should be via docker like the above command

Also should not include npm install. We can have a separate make command for that, but it should not be included when running `make jest` to avoid slowing down test feedback

Reply:
By using Docker, we avoid needing local PHP, NPM, etc, and associated potential version or config issues